scripts/generatenetworkssubgraphs.js

```diff
@@ -20,6 +20,14 @@ async function replaceContractAddresses() {
     }
     console.log('Creating subgraph.yaml for ' + network)
     let subgraph = fs.readFileSync('./subgraph.template.yaml', 'utf8')
+    const subgraphVe = fs.readFileSync('./subgraph_ve.template.yaml', 'utf8')
+    if (addresses[network].veOCEAN) {
+      // fix identation , due to vs auto format (subgraph_ve.template is moved to left)
+      const lines = subgraphVe.split('\n')
+      for (let line = 0; line < lines.length; line++) {
+        subgraph += '  ' + lines[line] + '\n'
+      }
+    }
```

Comment:
Thanks Alex, this "looks good" although crazy to see the need to fix indentation from the build.

Reply:
My VS Code will "fix" indentation for every save on subgraph_ve.yaml, making impossible to join them :)